dev_guide/builds/jenkins_pipeline_builds.adoc

```diff
@@ -0,0 +1,169 @@
+                +--------------------|   Pipeline build object has been created. A new Pipeline   |
+                |                    |   job is then created and a new Jenkins Slave is requested |
+                |                    |                                                            |
+                |                    +------------------------------------------------------------+
+                v
++------------------------------------------------------------+
+|                                                            |
+|   The content of the JenkinsFile is run on the Jenkins     |
+|   Slave pod. A new build on OpenShift is triggered via     |---------------------+
+|   The "Trigger a Build" command in the JenkinsFile         |                     |
+|                                                            |                     |
++------------------------------------------------------------+                     |
+                                                                                   v
+                                                                  +---------------------------------+
+                                                                  |                                 |
+                                                                  |   OpenShift runs the build as   |
+                +-------------------------------------------------|   normal and marks the Pipeline |
+                |                                                 |   build as complete             |
+                |                                                 |                                 |
+                |                                                 +---------------------------------+
+                v
++---------------------------------+
+|                                 |
+|   The Jenkins Master observes   |
+|   that the Pipeline build is    |
+|   complete and removes the      |
+|   Jenkins Slave pod             |
+|                                 |
++---------------------------------+
+----
+
+[[openshift-jenkins-Pipeline-plugin]]
+
+== OpenShift Jenkins Pipeline Plugin
+
+The https://github.com/openshift/jenkins-client-plugin[OpenShift Jenkins Pipeline Plugin] must be installed
+on your Jenkins slaves so that the OpenShift DSL will be available to process the JenkinsFile for your
+application.  This plugin is installed and enabled by default when using the Jenkins (Ephemeral) or Jenkins
+(Persistent) templates.
+
+For more information about installing and configuring this plugin, see xref:../install_config/configuring_pipeline_execution.html#openshift-pipeline-dsl-plugin[Installation and Configuration - Configuring Pipeline Execution].
+
+[[the-openshift-dsl]]
+
+== OpenShift DSL
+
+The OpenShift Jenkins Pipeline Plugin provides a Domain Specific Language (DSL) for communicating with the OpenShift
```

Comment:
client plugin.

the pipeline plugin is the legacy plugin that we are deprecating.

Reply:
updated in next push